src/genie/libs/parser/nxos/tests/test_show_eigrp.py

```diff
@@ -441,5 +442,324 @@ def test_show_eigrp_neighbors_detail_empty(self):
             parsed_output = obj.parse()
 
 
+class test_show_eigrp_topology(unittest.TestCase):
+
+    maxDiff = None
+    device = Device(name='aDevice')
+
+    device_output_empty = {'execute.return_value': ''}
+
+    expected_parsed_output_1 = {
+        "as": {
+            "1": {
```

Comment:
@tahigash please comment, does it need to be an integer?

Reply:
Yes! AS number is always an integer value.

Reply:
confirmed the change.